packages/front-end/enterprise/components/Dashboards/DashboardEditor/DashboardBlock/MetricBlock.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useMemo } from "react";
+import { MetricBlockInterface } from "back-end/src/enterprise/validators/dashboard-block";
+import { isDefined } from "shared/util";
+import { groupBy } from "lodash";
+import {
+  expandMetricGroups,
+  ExperimentMetricInterface,
+} from "shared/experiments";
+import useOrgSettings from "@/hooks/useOrgSettings";
+import ResultsTable from "@/components/Experiment/ResultsTable";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import { getMetricResultGroup } from "@/components/Experiment/BreakDownResults";
+import { BlockProps } from ".";
+
+export default function MetricBlock({
```

Comment:
rename to ExperimentMetricBlock (and change type field). Standalone metrics are one of the first expansions of dashboards I'd imagine we'll build